deploy/infra/aws/tf-deploy-infra/vars.tf

```diff
@@ -0,0 +1,74 @@
+variable "cp_account_id" {
+  description = "Please enter 12-digit AWS account ID"
+}
+
+variable "cp_region" {
+  description = "AWS region for deployment"
+  default = "us-east-1"
+}
+
+variable "cp_project" {
+  description = "The project name is substituted in the body of policies and service names"
+  default = "epam"
+}
+
+variable "cp_name_core" {
+  description = "Core site name."
+  default = "cloud-pipeline-core"
+}
+
+variable "cp_name_share" {
+  description = "Share site name"
+  default = "cloud-pipeline-share"
+}
+
+variable "cp_env" {
+  description = "Enviromnent. Used in tags"
```

Comment:
"Environment name. Used for tagging resources. E.g. 'dev' for development environment, or 'prod' for production one."